tests/new_functional/pages/surveys/question.page.js

```diff
@@ -19,6 +19,7 @@ class QuestionPage extends BasePage {
 
   submit() { return '[data-qa="btn-submit"]'; }
 
+  saveSignOut() { return '[data-ga-action="Save and complete later click"]'; }
```

Comment:
We should probably add a `data-qa` attribute to this button and use that instead of hooking on the the GA attribute?